src/hooks/useFirestoreConnect.ts

```diff
@@ -40,8 +60,9 @@ export interface SparkleRFQConfig extends ReduxFirestoreQuerySetting {
  * @see ValidFirestoreKeys
  * @see ReduxFirestoreQuerySetting
  */
-export const useSparkleFirestoreConnect = (config: SparkleRFQConfig[]) =>
-  useFirestoreConnect(config);
+export const useFirestoreConnect = (
+  config: (SparkleRFQSetting | SparkleDocRFQSetting)[]
+) => _useFirestoreConnect(config);
```

Comment:
Curious why the rename to shadow the default lib? IMO it makes it more confusing when I see it in code, I then have to mentally reason about/go check the imports, etc.